Source/WebCore/bindings/scripts/CodeGeneratorJS.pm

```diff
@@ -7441,8 +7441,8 @@ sub JSValueToNative
     push(@conversionArguments, $thisObjectReference) if JSValueToNativeDOMConvertNeedsThisObject($type);
     push(@conversionArguments, $globalObjectReference) if JSValueToNativeDOMConvertNeedsGlobalObject($type);
     push(@conversionArguments, $exceptionThrower) if $exceptionThrower;
-    if ($type->extendedAttributes->{StringContext}) {
-        my $interfaceName = $interface->type->name;
+    if ($type->extendedAttributes->{StringContext} || $type->name eq "ScheduledAction") {
```

Comment:
We have a goal of not further hardcoding any type names into the bindings script. Is there a way to achieve this with an attribute instead of a known name?